packages/wallet/src/components/dialogs/CreatePaymentPointerDialog.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button } from '@/ui/Button'
+import { Dialog, Transition } from '@headlessui/react'
+import { Fragment } from 'react'
+import { Select } from '@/ui/forms/Select'
+import type { DialogProps } from '@/lib/types/dialog'
+import { Form, useZodForm } from '@/ui/forms/Form'
+import { z } from 'zod'
+import { Input } from '@/ui/forms/Input'
+
+type CreatePaymentPointerDialogProps = Pick<DialogProps, 'onClose'> & {
+  account?: {
+    name: string
+    value: string
+  }
+}
+
+const createPaymentPointerSchema = z.object({
+  account: z.string().uuid(),
+  paymentPointer: z.string().min(3, {
+    message: 'Payment pointer should be at least 3 characters long.'
+  }),
+  publicName: z.string().min(3, {
+    message:
+      "Payment pointer's public name should be at least 3 characters long."
+  })
+})
+
+export const CreatePaymentPointerDialog = ({
+  onClose,
+  account
+}: CreatePaymentPointerDialogProps) => {
+  const form = useZodForm({
+    schema: createPaymentPointerSchema
+  })
+
+  const handleSubmit = form.handleSubmit((data) => {
+    console.log(data)
+  })
+
+  return (
+    <Transition.Root show={true} as={Fragment} appear={true}>
+      <Dialog as="div" className="relative z-10" onClose={onClose}>
+        <Transition.Child
+          as={Fragment}
+          enter="ease-out duration-300"
+          enterFrom="opacity-0"
+          enterTo="opacity-100"
+          leave="ease-in duration-200"
+          leaveFrom="opacity-100"
+          leaveTo="opacity-0"
+        >
+          <div className="fixed inset-0 bg-gradient-to-tr from-[#56B1AF]/90 via-[#92DBCA]/90 to-[#56B1AF]/90 transition-opacity" />
```

Comment:
should we add the gradient to config file? WOuldn't other dialogs use the same gradient?